openstack/sap-seeds/templates/_flavors_deleted.tpl

```diff
@@ -24,6 +24,25 @@
 #   ON j1.instance_uuid = i2.uuid
 #   WHERE flavor in ('$FLAVOR');
 
+- name: "m5.96xlarge"
+  id: "270"
+  vcpus: 90
+  ram: 1468416
+  disk: 64
+  is_public: false
+  extra_specs:
+{{- if .Values.use_hana_exclusive }}
+    {{- tuple . "vmware_hana_exclusive" | include "sap_seeds.helpers.extra_specs" | indent 4 }}
+{{- else }}
+    {{- tuple . "vmware_common" | include "sap_seeds.helpers.extra_specs" | indent 4 }}
+    "host_fraction": "1/4,3/4,1/2,1"
+    "resources:CUSTOM_BIGVM": "2"
+    "resources:CUSTOM_MEMORY_RESERVABLE_MB": "1468416"
+{{- end }}
+    "trait:CUSTOM_NUMASIZE_C48_M729": "required"
+    "hw:cpu_cores": "48"  # used in nova-vmware as cores-per-socket (24pCPU = 48vCPU)
```

Comment:
It says here `"hw:cpu_cores": "48"` vs `vcpus: 90`  in line 29, is that intended?

Reply:
see above.